myhpi/static/scss/footer.scss

```diff
@@ -1,5 +1,5 @@
 .footer-myhpi {
-    background-color: var(--bs-gray-700);
+    background-color: var(--bs-gray-800);
 
```

Comment:
Is there a way to increase the contrast? Firefox throws a contrast warning

![grafik](https://github.com/user-attachments/assets/7ef72582-c17a-436c-a5bf-1659fe7ee05f)